NEWS.md

```diff
@@ -1,3 +1,10 @@
+# New in snapd 2.59.1:
+
+* Add udev rules from steam-devices to steam-support interface
+* Bugfixes for layout path checking, dm_crypt permissions,
+  mount-control interface parameter checking, kernel commandline
+  parsing, docker-support, refresh-app-awareness
```

Comment:
nitpick; I would have spelled it: refresh app awareness  or refresh app-awareness